### PR TITLE
Portfolio Project - Projects Details Popup

### PR DESCRIPTION
**Pull request by mwismann**
### Portfolio Project - Projects Details Popup 😎 
**In this feature branch, my coding partners and I worked on the following features:**

- Created a function that allows the popup with details about the project to appear when the user clicks (or taps) the button to check project details,  ✅ 
- Created a function that makes the popup disappear when the user clicks (or taps) the close (X) button,  ✅ 
- Stored all the necessary information for the projects inside of Objects ✅
-  Stored all the Objects in an Array ✅

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -272,6 +313,71 @@ li {
...
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  text-align: left;
+  gap: 0.75rem;
+  background-color: white;
+  border-radius: 1rem;
+  z-index: 20;
+}
```
Comment (Qoosim) [inline styles.css]: - [x] It is required to stick to the template design as much as possible. Your pop up box is not properly positioned to the center of the page as we have it on the guiding template you chose. Kindly look at the both designs and work your design alongside the Figma template you chose. This applies to both the desktop and mobile versions.

### Your Desing Desktop
desktop

### Figma Design Desktop
temp-desktop


### Your Design Mobile
mobile

### Figma Design Mobile
temp-mobile

File: index.html [hunk 2]
```diff
@@ -97,7 +97,7 @@ <h3>Multi-Post Stories Gain + Glory</h3>
                             <li>html</li>
                         </ul>
     
-                        <a href="#" class="action-btn">See Project</a>
+                        <a href="#" class="action-btn" data-id="1">See Project</a>
                     </div>
                 </div>
                 <div class="works-grid-card">
```
Comment (Qoosim) [inline index.html]: - [ ] It is required that the projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for projects list should be removed, and instead it should be added via JavaScript. This is contained in the project requirements. Kindly take a look at the project requirements once again, remove already created html codes for the project sections and create them dynamically from your js file.

project-rqts

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BregornOriginal — APPROVED** [review]
## Hi @mwismann  

## Your project is complete!  There is nothing else to say other than... it's time to merge it 

Congratulations! :gorro_de_fiesta:

allfather-approves-approved

## Highlights

✅ Linters are passing
✅ Good commit history

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Pull request merged**